maps4fs/generator/texture.py

```diff
@@ -57,13 +57,15 @@ def __init__(  # pylint: disable=R0917
             width: int | None = None,
             color: tuple[int, int, int] | list[int] | None = None,
             exclude_weight: bool = False,
+            priority: int | None = 999,
```

Comment:
Set the default value as None.

Reply:
Maybe get rid of None here aand keep the default? Not sure how sorting would work otherwise.